src/main/scala/io/iohk/ethereum/db/dataSource/IodbDataSource.scala

```diff
@@ -32,4 +53,29 @@ object IodbDataSource {
   private def storageVersionGen(): Array[Byte] = {
     ByteBuffer.allocate(java.lang.Long.SIZE / java.lang.Byte.SIZE).putLong(updateCounter.incrementAndGet()).array()
   }
+
+  def apply(path: String, keySize: Int): IodbDataSource = {
+    //Delete directory if it existed and create a new one
+    val dir: File = new File(path)
+    val dirCreationSuccess = (!dir.exists() || deleteDirectory(dir)) && dir.mkdir()
+    assert(dirCreationSuccess, "Iodb folder creation failed")
+
+    val lSMStore: LSMStore = new LSMStore(dir = dir, keySize = keySize, keepSingleVersion = true)
```

Comment:
the doc says:
`  * @param keepSingleVersion  if true compaction will automatically delete older versions, it will not be possible to use rollback`
but as I see we use rollback in `clear`

am I missing something?